recipes/spirv-cross/config.yml

```diff
@@ -7,29 +13,5 @@ versions:
     folder: all
   "1.3.231.1":
     folder: all
-  "1.3.224.0":
-    folder: all
-  "1.3.216.0":
-    folder: all
-  "1.3.211.0":
-    folder: all
-  "1.3.204.0":
-    folder: all
-  "cci.20211113":
-    folder: all
-  "cci.20210930":
-    folder: all
-  "cci.20210823":
-    folder: all
-  "cci.20210621":
-    folder: all
-  "20210115":
-    folder: all
```

Comment:
Please keep this option as it is an active requirement of "shaderc" recipe at https://github.com/conan-io/conan-center-index/blob/91e67bdec23d59d4e2a104a3343762142b300058/recipes/shaderc/all/conanfile.py#L71